### ISSUE TITLE
Intro to Thimble UI Teaching Kit and Project

### ISSUE THREAD

**Issue by humphd** [mozilla/thimble.mozilla.org#1173]
From https://twitter.com/mozilla/status/651778087596785664, a _great_ suggestion to have some kind of learning artifact that helps people figure out how to use the UI.  Could we build some kind of teaching kit and remixable project on the homepage which is about learning how to use Thimble as such vs. how to make things on the web.

Some of what I have in https://github.com/mozilla/thimble.webmaker.org/wiki/Using-Thimble might be usable, or inspiration.  I bet people don't know about taking selfies, dragging files (images) in, using inline editors and docs, etc.  We should have a project + tutorial that shows them how!

I'm adding this to the `Pre MozFest` milestone with the hope that we can find some people to help make it happen.

cc @chadsansing, @hannahkane, @flukeout, @ldecoursy, @Pomax, @chrislarry33

**Comment by jgmac1106** [comment]
I can help. Been fooling around with some tutorials already.

My idea would be a basic "Hello world"  project that starts with a selfie
project and works up to downloading a few page "about me" website from a
git repo.

Within the tutorial.html file we highlight all the features. @humphd
shared.

That could be followed with a how "the heck to you use Git" tutorial that
has people download their new website and an upload it to their own repo
(tutorial I need).

**Comment by chadsansing** [comment]
@jgmac1106, let me know how to support the tutorial-making. I'll mull GitHub tutorials w/ you, too, further down the road, if that'd be helpful.

**Comment by jgmac1106** [comment]
Not sure if this messes up the pre-moszfest heartbeat but I won't have much
capacity until Oct 15th.

After that date I can build up the site. I think I am going to start with
migrating Doug's about me page from old Thimble to New Thimble:
https://dajbelshaw.makes.org/thimble/MTY5Mjk5MTQ4OA==/doug-belshaw-open-educational-thinkering
It
has always been approachable to n00bs and something they can wrap their
heads around.

This will be a walktrhough project that goes through all the Thimble
features. I will embed a bunch of gif and stuff.

Then I will make a "portfolio" template website on a git repo that people
have to download as a zip. I am thinking this will be the final for my
class this semester.

**Comment by jgmac1106** [comment]
Here was my first pass: https://d157rqmxrxj6ey.cloudfront.net/jgmac1106/25549/

I am going to start on the tutorial file but wanted to get an eye on this.

My major question is the use of the defined pixel size of the div box? Would that now be considered bad practice? Should it by percent or should the whole content div container a thing of the past and better practice is more a full width website?

**Comment by jgmac1106** [comment]
@chadsansing @humphd Here is my first attempt. https://d157rqmxrxj6ey.cloudfront.net/jgmac1106/25549/

I started on the activity page but thought for now this might be better as a stand alone tutorial. Please take a look and see if I hit all the features and didn't get any thing wrong.

I had a little trouble with the pre code class.

Though I wonder if this issue belongs in the Learning Repo. I will share on Discourse as well.

**Comment by chadsansing** [comment]
@jgmac1106, [shared feedback on Discourse] (https://discourse.webmaker.org/t/thimble-project-to-introduce-ui/180/2).

**Comment by jgmac1106** [comment]
@humphd Would it be bad form just to publish a project to host all of the images? Having that many files in the tree made navigation a little hard for users to navigate.

I am also going to experiment (has to wait until after my P&T binder) with making before and after images of just the code window so they will fit natively on most screens.

I forgot most of the world doesn't have a big display.

**Comment by jgmac1106** [comment]
@humphd @omnignorant @chadsansing 
I redid the tutorial. https://d157rqmxrxj6ey.cloudfront.net/jgmac1106/7423/

All of the image files are hosted in another project so the file tree doesn't get crazy and I used smaller gifs.

**Comment by humphd** [comment]
@jgmac1106 the link you have above doesn't look like the right thing.  Can you give the right link?

**Comment by jgmac1106** [comment]
Here you go. I remixed it as the number of images in the file were messing kids up. Sorry I missed the mention: https://thimble.mozilla.org/user/jgmac1106/49889

Here are more that @sedge  created: https://d157rqmxrxj6ey.cloudfront.net/sedge/12075/ and https://d157rqmxrxj6ey.cloudfront.net/sedge/12075/

**Comment by hannahkane** [comment]
I think @jgmac1106's project is a good intro project, similar to the Keep Calm make, but more personal for the user, which I like.

I also like what @sedge created, because it's very simple and straightforward, as well as pretty comprehensive. 

I'm wondering, though, if what we're wanting here is actually an in-product tour (similar to what we just did with Goggles). I just rewatched the YT video from the original tweet that prompted all of this, and I think he's asking for a high-level overview, rather than an in-the-weeds experience.
